curriculum/challenges/english/01-responsive-web-design/css-variables-skyline/part-115.md

```diff
@@ -9,17 +9,20 @@ dashedName: part-115
 
 Copy and paste your whole `sky` class along with all of its properties and values into the media query. You are going to make another color scheme for the skyline that changes it from day to night.
 
+Note: You are going to need to scroll past the editable region to copy the class.
+
 # --hints--
 
-test-text
+You should not delete the existing `.sky` declaration.
+
+```js
+assert.match(new __helpers.CSSHelp(document).getStyle(".sky")?.getPropVal('background', true), /radial-gradient\(rgb\(255,207,51\)|(0%),rgb\(255,207,51\)20%,rgb\(255,255,102\)21%,rgb\(187,238,255\)100%\)/);
```

Comment:
Not sure what is wrong here, but this test doesn't pass even with the starting code.